cognite/neat/rules/importers/_inference2rules.py

```diff
@@ -246,8 +246,8 @@ def _default_metadata(cls):
             created=datetime.now(),
             updated=datetime.now(),
             description="Inferred model from knowledge graph",
-            prefix="inferred",
-            namespace="http://purl.org/cognite/neat/inferred/",
+            prefix="neat",
+            namespace=DEFAULT_NAMESPACE,
```

Comment:
I had to do these two to make it work. Lets discuss that.

Reply:
![image](https://github.com/cognitedata/neat/assets/60234212/295d8fa1-f311-4e09-9f59-e6f86db44cc4)